workflow/rules/common.smk

```diff
@@ -396,7 +396,7 @@ def get_sample_datatype(sample):
 def get_markduplicates_input(wildcards):
     aligner = "star" if get_sample_datatype(wildcards.sample) == "rna" else "bwa"
     if sample_has_umis(wildcards.sample):
-        return "results/mapped/{aligner}/{{sample}}.annotated.bam".format(
+        return "results/mapped/{aligner}/{{sample}}.annotated.sorted.bam".format(
```

Comment:
Should we maybe make the type of `sorted` clear via the name? For example:

```suggestion
        return "results/mapped/{aligner}/{{sample}}.annotated.fgbio_queryname_sorted.bam".format(
```

Reply:
When using the piping functionality we can skip the sorting-suffix as I would write the pipe output the `pipe`-directory as we did for cutadapt.